ticketbuyer/purchase.go

```diff
@@ -31,7 +31,6 @@ func NewPurchaseManager(w *wallet.Wallet, purchaser *TicketPurchaser,
 
 // purchase purchases the tickets for the given block height.
 func (p *PurchaseManager) purchase(height int64) {
-	log.Infof("Block height %v connected", height)
```

Comment:
This was removed because it is redundant with dcrwallets output. Was only useful when ticketbuyer was its own app.